jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -157,12 +192,21 @@ private static boolean isProgressFooterEnabled(Project project) {
     }
   }
 
-  private final Project project;
-  private final SingleThreadedExecutor singleThreadedExecutor = new SingleThreadedExecutor();
-  private final ConsoleLogger consoleLogger;
+  @VisibleForTesting
+  @SuppressWarnings("java:S2065") // keep sonar happy. We cannot serialize an executor
+  transient SingleThreadedExecutor singleThreadedExecutor = new SingleThreadedExecutor();
+
```

Comment:
@wwadge I could definitely be missing something about the Gradle 8.x updates. Could you provide a bit of context regarding the use of a transient variable here?

Reply:
Got it, thanks for the explanation!